<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingForm</class>
 <widget class="QWidget" name="SettingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,4">
     <item>
      <widget class="QLabel" name="label_app_id">
       <property name="text">
        <string>app_id</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_app_id">
       <property name="text">
        <string>no setting</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,4">
     <item>
      <widget class="QLabel" name="label_api_key">
       <property name="text">
        <string>api_key</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_api_key">
       <property name="text">
        <string>no setting</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,4">
     <item>
      <widget class="QLabel" name="label_secret_key">
       <property name="text">
        <string>secret_key</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_secret_key">
       <property name="text">
        <string>no setting</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="pushButton_readSetting">
       <property name="text">
        <string>读取配置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_writeSetting">
       <property name="text">
        <string>保存配置</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_readSetting</sender>
   <signal>clicked()</signal>
   <receiver>SettingForm</receiver>
   <slot>pushButton_readSetting_OnClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>361</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_writeSetting</sender>
   <signal>clicked()</signal>
   <receiver>SettingForm</receiver>
   <slot>pushButton_writeSetting_OnClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>492</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel">
     <x>544</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>pushButton_readSetting_OnClick()</slot>
  <slot>pushButton_writeSetting_OnClick()</slot>
 </slots>
</ui>
